week-03/main.js

```diff
@@ -13,24 +14,40 @@ var list = [1, 2, 3, 4, 5];
 // and `hobbies` an array.
 // Assign the object to the variable `user`
 
-
+var user = {name: "Barrett", isAlive: true, hobbies: ["basketball", "video games"]};
 
 // Add 'golf' to the `user`'s `hobby` array.
-
+user.hobbies.push('golf');
 
 // # Functions
 
 // Define a function `sayHello` that takes a name string and returns "Hello" plus that name and an exclamation point.
 // Example:
 
 // sayHello('Mer') // "Hello Mer!"
+function sayHello(name) {
+  return "Hello" + name;
+}
 
 // Define a function `createRange` that takes `min` and `max` as parameters and returns an array with all the numbers
 // between min and max
 // Example:
 
 // createRange(4, 8) // [4, 5, 6, 7, 8]
-
+var arr = [];
```

Comment:
This function should return a new array that covers the range of numbers every time. Defining the array outside and accessing it globally means future calls will add more numbers to that array.